packages/compass-import-export/test/csv/many-columns.analyzed.json

```diff
@@ -0,0 +1,1312 @@
+{
```

Comment:
Is it worth having these text fixtures zipped like we do in `compass-e2e-tests`? https://github.com/mongodb-js/compass/blob/main/packages/compass-e2e-tests/package.json#L22
Looks like the `test` folder is getting close to 1mb already, might be more now, not sure what a threshold might be.

Reply:
I think its fine and we can file a ticket and deal with it if it ever becomes a problem. These files don't change much so I don't see it affecting day to day git operations.